src/rviz/ogre_helpers/stl_loader.cpp

```diff
@@ -91,6 +91,13 @@ bool STLLoader::load(const std::string& path)
 bool STLLoader::load(uint8_t* buffer)
 {
   uint8_t* pos = buffer;
+
+  // check for ascii type
+  if(std::string((char*)buffer,5) == std::string("solid")) {
```

Comment:
@caguero Maybe wording the error message is confusing, but there is an error if the STL file is an ASCII file. So we want to report an error if the file is in ASCII format, i.e. if the first 5 chars read as "solid".